Formula/openssh.rb

```diff
@@ -67,6 +68,10 @@ def install
       # Ensure sandbox profile prefix is correct.
       # We introduce this issue with patching, it's not an upstream bug.
       inreplace "sandbox-darwin.c", "@PREFIX@/share/openssh", etc/"ssh"
+
+      # FIXME: `ssh-keygen` errors out when this is built with optimisation.
+      # TODO: Report this upstream.
+      ENV.O0 if Hardware::CPU.intel? && MacOS.version == :ventura && version == Version.new("9.3p1")
```

Comment:
Left a `version` check here to try to make sure it's addressed at least at the next version bump.